packages/credential-status/src/__tests__/credential-status.test.ts

```diff
@@ -0,0 +1,73 @@
+import { createAgent } from '../../../core/src'
+import { CredentialStatusPlugin } from '../credential-status'
+import { DIDDocument } from 'did-resolver'
+
+describe('@veramo/credential-status', () => {
+  const referenceCredential = {
+    credentialStatus: {
+      type: 'ExoticStatusMethod2022',
+      id: 'some-exotic-id',
+    },
+  }
+  const referenceDoc = {} as DIDDocument
+
+  it('should check the credential status', async () => {
+    const expectedResult = {}
```

Comment:
It's probably a good idea to predefined the number of assertions since we're dealing with async methods.
```suggestion
    expect.assertions(3)
    const expectedResult = {}
```